spec/acceptance/trusted_external_data_spec.rb

```diff
@@ -119,4 +119,27 @@
       expect(trusted_json['external']['servicenow']['fqdn']).to eql(master.uri)
     end
   end
+
+  # This test provides coverage for oauth tokens both encrypted and not.
+  context 'user specifies a hiera-eyaml encrypted oauth token' do
+    # skip the oauth tests if we don't have a token to test with
+    servicenow_config = servicenow_instance.bolt_config['remote']
+    skip_oauth_tests = (servicenow_config['oauth_token']) ? false : true
+
+    let(:params) do
+      default_params = super()
+      default_params.delete(:user)
+      default_params.delete(:password)
+      default_params[:oauth_token] = master.run_shell("/opt/puppetlabs/puppet/bin/eyaml encrypt -s #{servicenow_config['oauth_token']} -o string").stdout
+      default_params
+    end
+
+    include_context 'setup cmdb'
+
+    it 'uses the new oauth token', skip: skip_oauth_tests do
```

Comment:
Out of curiosity, what does `rspec` output if the test is skipped?

Reply:
Nothing, it just says 0 failures, 1 pending

Reply:
We should probably print something that tells the user the test's being skipped and why it's being skipped (unless that's easy to figure out from rspec's current output, which seems like it isn't).